packages/market-service/package.json

```diff
@@ -20,6 +20,7 @@
     "test": "jest test"
   },
   "dependencies": {
+    "@shapeshiftoss/asset-service": "^1.0.0",
```

Comment:
uhhh we can't be doing this or we end up with coupled packages

Reply:
@cjthompson ?

Reply:
As long as there isn't a circular dependency it's fine.

Reply:
However, it does look to me like it doesn't actually do anything with asset-service.

Also, we should be using our DI model which would take an AssetService in the constructor on not have a direct dependency on the package.